tools/unsafe_optimizations.js

```diff
@@ -275,10 +275,7 @@ function runTests() {
   // Interaction between multiple passes:
   test('var d, f; f = new Uint8Array(16); var h = f.buffer; d = new Uint8Array(h);', 'var f=new Uint8Array(16),h=f.buffer,d=new Uint8Array(h);');
 
-  // Terser suboptimality! Should produce:
-  // test('var i=new Image;i.onload=()=>{}', 'var i=new Image;i.onload=()=>{}');
-  // but instead produces:
-  test('var i=new Image;i.onload=()=>{}', 'var i=new Image;i.onload=(()=>{});');
```

Comment:
What do you think about this part of the change?   Is it still useful?  What is it now testing?

Reply:
I think it's nice to verify that codegen is optimal there. I guess ideally this could just be in upstream terser, but since it's been an issue in the past, and it has almost no cost for us to test here, I'd keep it.

Reply:
OK, I'll a quick comment maybe